src/test/resources/VFBModel/GeppettoModelVFB.xmi

```diff
@@ -267,134 +267,145 @@
       dataSourceService="aberOWLDataSource"
       url="http://geppetto-unittests.virtualflybrain.org/api/runQuery.groovy"
       dependenciesLibrary="//@libraries.3"
+      targetLibrary="//@libraries.4"/>
+  <dataSources
+      id="owleryDataSourceSubclass"
+      name="owlery Data Source (subclasses)"
+      dataSourceService="owleryDataSource"
+      url="http://geppetto-unittests.virtualflybrain.org/kbs/vfb/subclasses"
+      dependenciesLibrary="//@libraries.3"
       targetLibrary="//@libraries.4">
-    <queries
-        xsi:type="gep_2:ProcessQuery"
-        name="Retains id, name and definition"
-        queryProcessorId="vfbAberOWLQueryProcessor"/>
     <queries
         xsi:type="gep_2:SimpleQuery"
-        name="Part of"
+        name="OWLery Part of"
```

Comment:
@ddelpiano minor...the spelling seems to be Owlery https://owlery.phenoscape.org/api/

Reply:
@tarelli again better for long-term support.